docker/lint.sh

```diff
@@ -37,6 +37,9 @@ function run_lint_step() {
             ;;
         asf)
             cmd=( tests/lint/check_asf_header.sh --local )
+            if [ $inplace_fix -eq 1 ]; then
```

Comment:
Can you edit the default case to explain what the `-i` flag does

Reply:
edited the --help